app/Models/System.php

```diff
@@ -79,12 +79,36 @@ protected static function newFactory(): SystemFactory
 
     // == constants
 
+    public const PlayStation = 12;
+    public const GameCube = 16;
+    public const Wii = 19;
+    public const WiiU = 20;
+    public const PlayStation2 = 21;
+    public const Xbox = 22;
+    public const AppleII = 38;
+    public const SegaSaturn = 39;
+    public const Dreamcast = 40;
     public const Arduboy = 71;
     public const WASM4 = 72;
     public const Uzebox = 80;
     public const Hubs = 100;
     public const Events = 101;
 
+    public static function getSystemsWithMultiDiscGames(): array
+    {
+        return [
+            System::PlayStation,
+            System::GameCube,
+            System::Wii,
+            System::WiiU,
+            System::PlayStation2,
+            System::Xbox,
+            System::AppleII,
+            System::SegaSaturn,
+            System::Dreamcast,
```

Comment:
rcheevos also supports m3u for CPC, C64, MSX, PC88, 3DO, PCE-CD, PCFX, and SegaCD.

Recommend sorting this alphabetically to make it easier to look at.

Reply:
Ripped out per https://github.com/RetroAchievements/RAWeb/pull/2582#issuecomment-2257082062.